jekyll/_cci2/orbs-faq.md

```diff
@@ -9,164 +9,66 @@ order: 20
 
-* Question: Why do I get the following error when testing locally:
+## Report an issue with an orb
 
-```
-circleci build -c .circleci/jobs.yml --job test
-```
+* **Question:** How can I report a bug or issue with an orb?
 
-```
-Error:
-You attempted to run a local build with version 2.1 of configuration.
-```
+* **Answer:** All orbs are open source projects. Issues, bug reports, or even pull requests can be made against the orb's git repository. Orb authors may opt to include a link to the git repo on the Orb Registry.
 
-* Answer: To resolve this error, run `circleci config process` on your configuration and then save that configuration to disk. You then should run `circleci local execute` against the processed configuration.
+  If the git repo link is unavailable, contact support and we will attempt to contact the author. Alternatively, consider forking the orb and publishing your own version.
 
-## Rerun Error
+## Using uncertified orbs
 
-* Question: Why do I get the following error when re-running the same workflow:
+* **Question:** Why do I receive an error message when trying to use an uncertified orb?
 
-```
-only certified orbs are permitted in this project.
-```
+* **Answer:** To enable usage of _uncertified_ orbs, go to your organization's settings page, and click the _Security_ tab. Then, click yes to enable _Allow Uncertified Orbs_.
 
-* Answer: Try making a whitespace change or similar. Your configuration will not recompile until you have made a change. Configuration processing occurs before the compiled code is passed into the workflows conductor. Because of that, the workflows conductor (where you trigger the rebuild) knows nothing of the original 2.1 config.
+**Note:** _Uncertified orbs are not tested or verified by CircleCI._
 
-## Environment Variables Not Being Passed at Runtime
+## How to use the latest version of an orb
 
-Occasionally, when you try to convert a configuration to a 2.0 compatible format, environment variables may not be passed at runtime. For example, if you create a simple configuration in your GitHub repository (for example `https://github.com/yourusername/circle-auto/blob/master/.circleci/echo.yml`) and then call the config using:
+* **Question:** How do import an orb always at the latest version?
 
-```
-export AUTO_FILE=/Users/yourusername/Desktop/apkpure_app_887.apk
-export AUTO_DIR=.
-circleci build -c .circleci/echo.yml --job test
-```
+* **Answer:** Orbs utilize [semantic versioning](), meaning if you set the _major_ version (example: `3`), you will receive all _minor_ and _patch_ updates, where if you statically set the version (example: `3.0.0`), no updates will apply, this is the most deterministic and recommended method.
```

Comment:
Do we want to link here again like in the intro? https://semver.org/

Reply:
Oops. Likely yes, but I was actually going to also bring up, what we want to do with the orb concepts page, which also goes over semver.

Reply:
Rather than linking to semver.org, lets link to the orb concepts page. Pushing that soon. Lets merge everything then, and there will be some links that will need to be updated. Ill do another pass then before moving on to more content

Reply:
🎉